pytest_splunk_addon/standard_lib/data_models/Network_Traffic.json

```diff
@@ -287,7 +305,7 @@
         },
         {
           "name": "src_translated_port",
-          "type": "conditional",
+          "type": "optional",
```

Comment:
I see it was conditional and the condition is actually good, so please keep as is.

Reply:
This is marked as optional in sheet https://docs.google.com/spreadsheets/d/1ZFDC0Efn-bHvcU1Qy78s95GCfWyxt6IUhTv94j3yagk/edit?gid=1499472344#gid=1499472344 hence the change.

Reverted it to conditional

Reply:
Yes, I corrected it in the sheet. Thanks.